utils/format/exceptions.hpp

```diff
@@ -56,6 +56,7 @@ class bad_format_error : public error {
     explicit bad_format_error(const std::string&, const std::string&);
     virtual ~bad_format_error(void) throw();
 
+    /// \return The format string that caused the error.
```

Comment:
I think we currently document all functions in the cpp file, not in the hpp. Move there. Same everywhere else.

Reply:
Okey dokey!